relayer/compose-zec.yaml

```diff
@@ -0,0 +1,34 @@
+name: relayer-zcash
+services:
+  zcashd:
+    image: electriccoinco/zcashd:latest
+    command:
+      - -printtoconsole
+      - -rpcuser=zcash
+      - -rpcpassword=zcash
+      - -rpcallowip=0.0.0.0/0
+      - -rpcbind=0.0.0.0
+      - -prune=550          # 550 MB target for block files
+      - -txindex=0          # prune mode is incompatible with txindex
+      # -testnet             # uncomment if you want testnet (uses ports 18232/18233)
+    ports:
+      - "8232:8232"         # RPC (18232 on testnet)
+      - "8233:8233"         # P2P (18233 on testnet)
+    volumes:
+      - zec-data:/home/zcash/.zcash
+      - ./configs/zcash.conf:/srv/zcashd/.zcash/zcash.conf
+  relayer:
```

Comment:
Looking at differences between this definition and btc/ltc, I see there is no platform field. Is it because zcash image is not multi-platform?